test/unit/test_amqpclient.js

```diff
@@ -685,16 +210,16 @@ describe('AMQPClient', function() {
 
 
     it('should create connection, session, and link on receive with full address', function() {
```

Comment:
This is no longer true. Should consider either renaming, or just drop the test if it's covered by other things.